README.md

```diff
@@ -24,21 +24,21 @@ We will install some dependencies before configuring OpenSwoole.
 #!/bin/bash
```

Comment:
The block was marked with a shebang so we didnt include $. If we are adding $ to it, its better we remove the shebang and merge all the `apt install` into single command and adding $ in front will make much more sense.

Reply:
Thank you for the notification! I have committed the changes, kindly check it!